api/odl2.py

```diff
@@ -174,7 +174,11 @@ def _extract_publication_metadata(self, feed, publication, data_source_name):
             skipped_license_formats = configuration.skipped_license_formats
 
             if skipped_license_formats:
-                skipped_license_formats = set(json.loads(skipped_license_formats))
+                skipped_license_formats = (
+                    set(json.loads(skipped_license_formats))
+                    if isinstance(skipped_license_formats, str)
+                    else set(skipped_license_formats)
```

Comment:
This implies that we have this config might be stored in the database in different formats. Is there any reason why we wouldn't want to enforce a single format for these? Alternatively, could we have a configuration getter for these ensures that that it always comes out the same way?

Wrapping the `set` around the entire ternary makes it a little more obvious that it's a similar issue as in `api/saml/provider.py` +88 to +92:
```
set(
  json.loads(skipped_license_formats))
  if isinstance(skipped_license_formats, str)
  else skipped_license_formats
)
```

Reply:
I added additional changes to https://github.com/ThePalaceProject/circulation-core/pull/35 in order to correctly parse JSON-encoded `LIST` and `MENU` items directly in `ConfigurationMetadata.__get___` and removed the code above.